src/analytics/private/sqls/redshift/item-v2.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE IF NOT EXISTS {{schema}}.item_v2 (
+    event_timestamp timestamp not null,
+    event_id varchar(255) not null,
+    event_name varchar(255) not null,
+    platform varchar(255),
+    user_pseudo_id varchar(255) not null,
+    user_id varchar(255),
+    item_id varchar(255) not null,
+    name varchar(255),
+    brand varchar(255),
+    currency varchar(255),
+    price DOUBLE PRECISION,
+    quantity DOUBLE PRECISION,
+    creative_name varchar(255),
+    creative_slot varchar(255),
+    location_id varchar(255),
+    category varchar(255),
+    category2 varchar(255),
+    category3 varchar(255),
+    category4 varchar(255),
+    category5 varchar(255),
+    custom_parameters_json_str varchar(65535),
+    custom_parameters super,
+    process_info super,
+    created_time timestamp DEFAULT getdate()
```

Comment:
it seems not contains the preset event attributes
https://github.com/awslabs/clickstream-web/blob/main/src/provider/Event.ts#L73-L108
 for `_page_load`